Resources/Locale/en-US/administration/antag.ftl

```diff
@@ -2,4 +2,5 @@ verb-categories-antag = Antag ctrl
 admin-verb-make-traitor = Make the target into a traitor.
 admin-verb-make-zombie = Zombifies the target immediately.
 admin-verb-make-nuclear-operative = Make target a into lone Nuclear Operative.
-admin-verb-make-pirate = Make the target into a pirate. Note this doesn't configure the game rule.
+admin-verb-make-pirate = Make the target into a pirate. Note that this doesn't configure the game rule.
+admin-verb-make-space-ninja = Make the target into a space ninja. Note that this doesn't configure the Traitor game rule.
```

Comment:
```suggestion
admin-verb-make-space-ninja = Make the target into a space ninja. Note that this doesn't configure the space ninja game rule.
```

Reply:
space ninja only has spawn game rule, the note is since if you dont enable the traitor gamerule you get no greentext
will clarify it